install/docker-compose/docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: '2'
+
+services:
+  mongo:
+    image: mongo:3.4.23
+    container_name: microcks-db
```

Comment:
Could you change to the old name that is `microcks-mongo`? That way we know it's the container attached to the `mongo` alias.

Reply:
This name is shown only when interacting with docker like in `docker ps` the alias used in the network level is still `mongo`. Is there a specific need to expose the implementation detail on db and sso ?

Reply:
I use to have many mongo containers running or in the `docker ps` history and love to see that one is coming from microcks. But I'll change it afterwards. No problem ;-)